packages/web/components/Dashboard/Post/PostComments.tsx

```diff
@@ -210,19 +212,24 @@ const PostComments = ({
           font-style: italic;
         }
 
-        .new-comment-block {
-          border-top: 1px solid ${theme.colors.gray400};
-          margin-top: 5px;
+        fieldset {
+          padding: 0 5px 5px;
+          border: 1px solid ${theme.colors.gray400};
+          border-radius: 5px;
+          background-color: ${theme.colors.gray100};
         }
 
         .new-comment-block :global(textarea) {
           min-height: 4em;
           width: 100%;
           padding: 5px;
-          background-color: transparent;
+          background-color: white;
```

Comment:
(testing a suggestion)
```suggestion
          background-color: ${theme.colors.white};
```

Reply:
Test